src/SharedLibrary/IntegrationTests/SharedLibrary_SSWFeaturizer_IntegrationTest.cpp

```diff
@@ -0,0 +1,84 @@
+// ----------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+// ----------------------------------------------------------------------
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+#include "GeneratedCode/SharedLibraryTests_StandardScaleWrapperFeaturizer.h"
+
+TEST_CASE("int8_t_with_mean_with_std") {
+    StandardScaleWrapperFeaturizer_int8_Test(
+        std::vector<std::int8_t>{
+            static_cast<std::int8_t>(0),
+            static_cast<std::int8_t>(0),
+            static_cast<std::int8_t>(1),
+            static_cast<std::int8_t>(1)
+        },
+        std::vector<std::int8_t>{
+            static_cast<std::int8_t>(2)
+        },
+        [](std::vector<std::double_t> const &args) {
+            return args == std::vector<std::double_t>{static_cast<std::double_t>(3)};
+        },
+        bool(true),

```

Comment:
> bool(true) [](start = 7, length = 11)

What does this mean?

Reply:
These two bools are configuration params for this featurizer which indicate should we use mean and standard deviation of input data.

Reply:
Do they stop being bool if you wrote true or false?

---
In reply to: [366533146](https://github.com/microsoft/FeaturizersLibrary/pull/65#discussion_r366533146) [](ancestors = 366533146)